samples/excel/pivot-tables/refresh-pivot-table.yaml

```diff
@@ -0,0 +1,166 @@
+id: ''
+gist: ''
+source: EXCEL
+author: Microsoft
+name: Refresh pivot table
+description: |
+  Refresh pivot table
+script:
+  content: |+
+    $('#create-table').click(createTable);
+    $('#refresh-pivot-table').click(refreshPivotTable);
+
+    /** Create a new table with some sample data */
+    function createTable() {
+        Excel.run(async (context) => {
```

Comment:
try {
		await OfficeHelpers.Excel.deleteWorksheetIfExists("Playground")
		await Excel.run(async (context) => {
			// Queue a command to add a new worksheet
			sheet = context.workbook.worksheets.add("Playground");

			// Call the helper to create the table
			queueCreateSampleTable(sheet);

			//Queue a command to activate the sheet
			sheet.activate();

			// Run all of the queued up commands
			await context.sync();
		});
	}
	catch (error) {
		OfficeHelpers.Utilities.log(error);
	}